core/src/test/java/org/commonjava/maven/ext/core/impl/ProjectVersionManipulatorTest.java

```diff
@@ -651,8 +651,8 @@ private TestVersioningModifier newVersioningModifier()
                 if ( applyVersioningChanges( new Project ( project.getOriginalModel()), state ) )
                 {
                     final String v = _versionsByGAV.get( SimpleProjectVersionRef.parse( gav( project ) ) );
-                    logger.info( project.getName() + " (" + gav( project ) + "): VERSION MODIFIED\n    New version: "
-                        + v );
+                    logger.info( "{} ({}): VERSION MODIFIED{}    New version: {}",
+                        project.getName(), gav( project ), System.lineSeparator(), v );
```

Comment:
Given this part hasn't been changed since 2014/5 .... I'd be tempted just to remove the logger line. We don't need it in tests (and if we do have logging / `System.out.println` I tend to use them in conjunction with https://stefanbirkner.github.io/system-rules/#SystemErrAndOutRule and mute for success.